### PR TITLE
Add important missing part of README.

### PR DESCRIPTION
**Pull request by ksbrar**
The world must know who authored this beautiful repo.

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,4 +1,4 @@
 molec
 =====
 
-Noble gas simulation using the Lennard-Jones potential
+Noble gas simulation using the Lennard-Jones potential by **Nickolas Dickolas of Penisshire**.
```
Comment (aristocrates) [inline README:4]:
```suggestion
Noble gas simulation using the Lennard-Jones potential.
```

File: README [hunk 2]
```diff
@@ -1,4 +1,4 @@
 molec
 =====
```
Comment (aristocrates) [inline README]:
```suggestion
```

File: README [hunk 3]
```diff
@@ -1,4 +1,4 @@
 molec
```
Comment (aristocrates) [inline README]:
```suggestion
# molec
```

File: README [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
 =====
```
Comment (aristocrates) [inline README:3]:
```suggestion
## Nicholas Meyer

```

**Review by aristocrates — CHANGES_REQUESTED** [review]

File: README (hunk 1 above)
Comment (ksbrar) [reply README:4]: I strongly disagree with this change.

File: README (hunk 4 above)
Comment (ksbrar) [reply README:3]: Redundant. Sad!

**Comment by ksbrar** [comment]
Reconsider your changes, Sir Nicholas.

**Comment by ksbrar** [comment]
Disappointed in the slow response times for this project. Closing my PR and taking it somewhere else. 0/10 even with rice.

**Pull request closed without merging**